CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 0.10.0 - 2010-11-06
```

Comment:
```suggestion
## Unreleased

## 0.10.0 - 2010-11-05
```

Reply:
I didn't do this since we are terrible about updating our unreleased-changelog entries. I figured we could add it if we actually put some entries in there.

Reply:
I added an `Unreleased` section with post-0.10.0 entries, so 👍